CryptoAnalysis-Android/src/main/java/de/fraunhofer/iem/crypto/CogniCryptAndroidAnalysis.java

```diff
@@ -143,16 +143,16 @@ private void prepareAnalysis() {
         logger.info("Library classes: "+ Scene.v().getLibraryClasses().size());
     }
 
-	protected List<CryptSLRule> getRules() {
-		List<CryptSLRule> rules = Lists.newArrayList();
+	protected List<CrySLRule> getRules() {
+		List<CrySLRule> rules = Lists.newArrayList();
 		if (rulesDirectory == null) {
 			throw new RuntimeException(
-					"Please specify a directory the CrySL rules (.cryptslbin Files) are located in.");
+					"Please specify a directory the CrySL rules (.cryslbin Files) are located in.");
 		}
 		File[] listFiles = new File(rulesDirectory).listFiles();
 		for (File file : listFiles) {
-			if (file != null && file.getName().endsWith("cryptslbin")) {
-				rules.add(CryptSLRuleReader.readFromFile(file));
+			if (file != null && file.getName().endsWith("cryslbin")) {
```

Comment:
Definitely use static constant from CrySLModelReader.java here. 
*Note: It's OK to use the dot '.' character with endsWith(string)*